src/instructions/mod.rs

```diff
@@ -10,9 +10,34 @@ pub mod tables;
 pub mod tlb;
 
 /// Halts the CPU until the next interrupt arrives.
-#[inline(always)]
+#[inline]
 pub fn hlt() {
     unsafe {
         asm!("hlt" :::: "volatile");
     }
 }
+
+/// Emits a 'magic breakpoint' instruction for the [Bochs](http://bochs.sourceforge.net/) CPU
+/// emulator. Make sure to set `magic_break: enabled=1` in your `.bochsrc` file.
```

Comment:
Could you add a link to https://wiki.osdev.org/Bochs#Magic_Breakpoint and/or http://bochs.sourceforge.net/doc/docbook/user/bochsrc.html#AEN2422 heer?